.circleci/config.yml

```diff
@@ -208,6 +209,7 @@ workflows:
             branches:
               only: release
       - deploy-to-staging:
+          context: api-nuget-token-context
```

Comment:
You would also need to add the context on the deploy-to-production job

Reply:
Great point, thanks! Added